WORKSPACE

```diff
@@ -109,4 +112,23 @@ maven(artifacts)
 load("@graknlabs_bazel_distribution//common:rules.bzl", "workspace_refs")
 workspace_refs(
     name = "graknlabs_protocol_workspace_refs"
-)
\ No newline at end of file
+)
+
+#########################
+# Load NPM dependencies #
+#########################
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "f2194102720e662dbf193546585d705e645314319554c6ce7e47d8b59f459e9c",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/2.2.2/rules_nodejs-2.2.2.tar.gz"],
+)
+
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "npm_install")
+node_repositories()
+npm_install(
+  name = "npm",
+  package_json = "//:package.json",
+  package_lock_json = "//:package-lock.json",
+)
```

Comment:
Performing an npm_install is required in order to get said node modules up to date and in the build, and we need to set up the nodejs rules so that we can fetch them in. (The node modules specifically required are the protoc compiler and the gen_grpc plugin for it)